src/citrine/_rest/engine_resource.py

```diff
@@ -75,36 +73,3 @@ class VersionedEngineResource(EngineResource[Self], IncludeParentProperties[Self
     def build(cls, data: dict):
         """Build an instance of this object from given data."""
         return super().build_with_parent(data, __class__)
-
-
-class ModuleEngineResource(EngineResource[Self], IncludeParentProperties[Self]):
```

Comment:
Is there no way to deprecate a class?  I thought I'd worked out a method that issued a warning on import.

Reply:
I'm pretty sure your class came after this, and was never added to it. Regardless, I imagine it would have caused using `DesignSpace` to throw deprecation warnings, which wasn't accurate.